starkjs/src/compressor12/main_compressor12_exec.js

```diff
@@ -42,9 +47,11 @@ async function run() {
     const pil = await compile(F, pilFile, null, pilConfig);
     await fs.promises.writeFile(pilFile+ ".json", JSON.stringify(pil, null, 1) + "\n", "utf8");
 
+// 1. pil -> cm
     const cmPols = newCommitPolsArray(pil);
-
+//2. wasm -> wc
     const wc = await WitnessCalculatorBuilder(wasm);
```

Comment:
here you can use the witness calculator from https://github.com/0xEigenLabs/eigen-zkvm/tree/main/plonky/src/witness